packages/jss-plugin-syntax-global/src/index.test.js

```diff
@@ -16,6 +16,25 @@ describe('jss-global', () => {
     jss = create(settings).use(global())
   })
 
+  describe('@global rule with null, undefined or empty value', () => {
```

Comment:
Can you also test `@global a` with `undefined` rule?

Reply:
I guess b: undefined should cover it?

Reply:
What @HenriBeck describes is  a prefixed rule, what you are testing is a container rule:
```js
{
  '@global a': {color: 'red'}
}
```

Reply:
Test it on sanbox, it's working.  

export default {
  '@global a': {color: 'red'}
};

Although not existing issue, I added it as a regression test in the PR.

Reply:
oh, wait. @HenriBeck , what the "prefixed rule" test case looks like?

Reply:
```js
'@global a': undefined
'@global b': null,
```

it might be already working, just to make sure we cover that case.

Reply:
it is working. I added those test cases as well to protect more such empty value edge cases.